app/views/admin/tags/show.html.erb

```diff
@@ -86,7 +86,7 @@
       </div>
       <div class="form-group">
         <%= f.label :text_color_hex %>
-        <%= f.color_field :text_color_hex, class: "form-control", required: true %>
+        <%= f.color_field :text_color_hex, class: "form-control", required: true, value: @tag.text_color_hex.nil? ? "#ffffff" : @tag.text_color_hex %>
```

Comment:
**(question, non-blocking)**: I didn't trace through the code but can @tag.text_color_hex be blank at any point? If it can then perhaps we should use blank? instead to check for nil, false, empty, or a whitespace string.

Reply:
Great suggestion! I have made this change @Ridhwana 😊